deploy/compliance-operator/config.yml

```diff
@@ -0,0 +1,9 @@
+deploymentMode: "SelectorSyncSet"
+selectorSyncSet:
+  matchLabels:
+  - key: api.openshift.com/environment
+    operator: NotIn
+    values: ["production", "integration"]
```

Comment:
Since I'm assuming the legal entity is only on stage, is there a reason this can't be

```
operator: In
values: ["stage"]
````

Reply:
good point. This makes more sense thanks